pyproject.toml

```diff
@@ -1,26 +1,31 @@
 [tool.poetry]
 name = "download-vertical-profiles"
-version = "0.4.0"
-description = ""
+version = "0.5.0"
+description = "Download .map, .vmr and .mod files from ccycle.gps.caltech.edu"
```

Comment:
Could you rephrase this line as "Download vertical profiles (GGG2014 and GGG2020) from ccycle.gps.caltech.edu"?